index.html

```diff
@@ -0,0 +1,424 @@
+              <ul>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">Ruby on Rails</a></li>
+                <li><a href="#">RSPec</a></li>
+                <li><a href="#">CapyBara</a></li>
+                <li><a href="#">Selenium</a></li>
+              </ul>
+            </div>
+          </div>
+
+          <div class="skills-heading">
+            <ul>
+              <li><a href="#">Skills</a></li>
+            </ul>
+
+            <div class="skill-list-items">
+              <ul>
+                <li><a href="#">Codekit</a></li>
+                <li><a href="#">Github</a></li>
+                <li><a href="#">Codepen</a></li>
+                <li><a href="#">Gitlab</a></li>
+                <li><a href="#">Terminal</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="contact-section" id="contact">
+        <div class="contact-heading">
+          <h3>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h3>
+        </div>
+        <div class="form-container">
+          <form action="https://formspree.io/f/xrgvbqyj" method="post" class="form">
+            <input type="text" name="name" id="name" placeholder="Full name" maxlength="30" required/>
+            <input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="Email address" required
+            />
+            <textarea
+              name="textarea"
+              id="textarea"
+              cols="5"
+              rows="5"
+              placeholder="Enter text here" maxlength="500" required
+            ></textarea>
+            <button class="form-button">Get in touch</button>
+          </form>
+        </div>
+      </section>
+
+      <footer class="footer-container">
+        <div class="footer-icon-list">
+          <ul>
+            <li>
+              <a href="#"
+                ><i class="fa fa-github footer-icon" aria-hidden="true"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fa fa-linkedin footer-icon" aria-hidden="true"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fa fa-angellist footer-icon" aria-hidden="true"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fa fa-twitter footer-icon" aria-hidden="true"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#" class="footer-vectorimg"
+                ><img
+                  src="./images/footericon.png"
+                  class="footer-vectorimg"
+                  alt="Vector"
+              /></a>
+            </li>
+          </ul>
+        </div>
+        <div class="border-line"></div>
+      </footer>
```

Comment:
- [ ] [OPTIONAL] You have done very great work on implementing the line 👏 👏 . However,  I have noticed that the line does not display as intended. Kindly ensure that the line is displayed in a proper way that matches the Figma design 👌

Your design 😎 
 
![noline](https://user-images.githubusercontent.com/99394374/213869076-2eadc898-eea1-4406-b9e8-6af7da6f555c.PNG)

Figma design 😎 
![nolin](https://user-images.githubusercontent.com/99394374/213869091-fe035392-c6a9-4590-8f98-b1fcb7388457.PNG)